extobjc/metamacros.h

```diff
@@ -6,26 +6,32 @@
  * Released under the MIT license
  */
 
-#ifndef EXTC_METAMACROS_H
-#define EXTC_METAMACROS_H
-
+/// Two things have been done in order to override macros from other versions of this file:
```

Comment:
Put this in the commit description and remove it from the source (I don't think it belongs here).